spring-cloud-gcp-kms/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Are you actually relying on this?

Reply:
Yes due to mocking the final methods of the SDK